org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/impl/javasupport/JdtToBeBuiltComputer.java

```diff
@@ -8,40 +8,53 @@
 package org.eclipse.xtext.builder.impl.javasupport;
```

Comment:
wonder if this should fix the tests in IntegrationTest and SimpleProjectsIntegrationTest that are ignored through https://github.com/eclipse/xtext-eclipse/issues/400

at least the IntegrationTest are still constantly red

Reply:
I hope so.